wrappers/ortools.rb

```diff
@@ -465,19 +465,6 @@ def kill
 
     private
 
-    def build_cost_details(cost_details)
```

Comment:
Here, the object cost_details is provided by the protobuf model returned by OR-Tools. Its parsing is specific to this solver

Reply:
If we remove this function then we should remove it from vroom.rb too. But as @braktar said maybe it is not possible. 

I think we can have minimal build_cost_details in wrapper.rb and then overlad it and fill each key according to each solver specificities.

Reply:
Wrapper.rb should implement the method and the solver wrappers should override it if necessary

Reply:
It turns out the solution was right in front of us... `Models::CostDetails.new({})` and we don't even need a `build_cost_details` function for `empty_result` generation.

By the way, I checked your suggestion and it doesn't work. Even if we have the function defined in `Wrappers::Wrapper` and `Wrappers::Ortools` since the empty_result is called via `config[:services][service].empty_result` -- i.e., from `Wrappers::Ortools` -- due to inheritance it behaves as a `Wrappers::Ortools` function (even though empty_result is only defined in `Wrappers::Wrapper`) and it calls the `build_cost_details` of `Wrappers::Ortools` in this case, instead of the vanilla version defined in `Wrappers::Wrapper`. Since all of our methods are instance methods I don't see a way around this without a design overhaul.